Sources/ZcashLightClientKit/Block/Scan/BlockScanner.swift

```diff
@@ -57,7 +57,8 @@ extension BlockScannerImpl: BlockScanner {
             let scanSummary: ScanSummary
             let scanStartTime = Date()
             do {
-                let fromState = try await service.getTreeState(BlockID(height: startHeight - 1))
+                // ServiceMode to resolve
+                let fromState = try await service.getTreeState(BlockID(height: startHeight - 1), mode: .defaultTor)
```

Comment:
This should be 
```suggestion
                // ServiceMode to resolve
                // Directly correlated with `BlockDownloader` ranges.
                let fromState = try await service.getTreeState(BlockID(height: startHeight - 1), mode: .defaultTor)
```
as it is directly correlatable with the `BlockDownloader` calls (every block range downloaded has a corresponding `getTreeState` call), and thus can be fetched the same way.